.github/workflows/example-reusable.yml

```
@@ -1,14 +1,20 @@
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+# This is an example workflow demonstrating how to use the reusable workflows
+# defined in .github/workflows/
+
+on:
```

YAMLlint (1.37.1)

[error] duplication of key "on" in mapping

(key-duplicates)